robusta_krr/core/integrations/rollout.py

```diff
@@ -1,10 +1,7 @@
 from kubernetes import client
```

Comment:
is this from a specific version of the kubernetes cli?
Any reason why we dont just upgrade the kubernetes cli to the relevant version instead of having this file?